src/module/display.js

```diff
@@ -40,10 +52,12 @@ export default class Interact {
 
   updateTodo = (e) => {
     const idUpdate = e.target.id;
-    const newValue = document.querySelector('.new-input').value;
-    updateTask(idUpdate, newValue);
+    const input = document.querySelector('.new-input');
+    const newValue = input.value;
+    const m = this.lists.find((todo) => Number(todo.index) === Number(idUpdate));
+    m.description = newValue;
+    savedTodos(this.lists);
```

Comment:
- [optional] When I clicked on the edit icon, I observed that the text value is cleared off. Kindy address this for a great user experience. 👍 

## Your App
 
<img width="339" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/221378347-95d9de8e-e3a9-4cd0-9f81-50e5be446640.png">